package.json

```diff
@@ -36,6 +36,7 @@
     "build:desktop": "./bin/package-electron",
     "build:api": "yarn workspace @actual-app/api build",
     "generate:i18n": "yarn workspace @actual-app/web generate:i18n",
+    "generate:release-notes": "npx tsx ./bin/release-note-generator.ts",
```

Comment:
One observation: I could imagine folks not having `npx` installed, but `ts-node` is already a dependency of `loot-core` so it will be. Could we just run with `ts-node` directly?

Reply:
https://www.npmjs.com/package/npx Per the npx package page, npx is now part of npm. Let me know if my assumption is wrong because I don't remember installing npx explicitly on my machine

Reply:
While it's true that `npx` is part of `npm` nowadays, the project uses `yarn` and I _believe_ it's not included with `yarn` if you install via a package manager

Reply:
You are right! I installed ts-node as a dev dependency in the base actual package.json and switched it over.

Reply:
Awesome, thank you!